iothub/device/samples/readme.md

```diff
@@ -1,27 +1,71 @@
-Device and module samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+
+This folder contains simple samples showing how to use the various features of Microsoft Azure IoT Hub service, from a device running C# code.
 
 ### [Device samples][device-samples]
-* [DeviceClientHttpSample][d-message-sample]
-* [DeviceClientAmqpSample][d-message-sample]
-* [DeviceClientMqttSample][d-message-sample]
-* [DeviceClientFileUploadSample][d-file-upload-sample]
-* [DeviceClientKeysRolloverSample][d-keys-rollover-sample]
-* [DeviceClientMethodSample][d-method-sample]
-* [DeviceClientTwinSample][d-twin-sample]
-* [XamarinSample][d-xamarin-sample]
+* [Message Sample][d-message-sample]
+* [Method Sample][d-method-sample]
+* [Twin Sample][d-twin-sample]
+* [File Upload Sample][d-file-upload-sample]
+* [Keys Rollover Sample][d-keys-rollover-sample]
+* [Import Export Devices Sample][d-import-export-devices-sample]
+* [Plug and Play Device Sample][d-pnp-sample]
+* [Xamarin Sample][d-xamarin-sample]
 
 ### [Module samples][module-samples]
-* [MessageSample][m-message-sample]
-* [TwinSample][m-twin-sample]
+* [Message Sample][m-message-sample]
+* [Twin Sample][m-twin-sample]
+
+### Prerequisites
+In order to run the device samples on Linux or Windows, you will first need the following prerequisites:
+* [Setup your IoT hub][lnk-setup-iot-hub]
+* [Provision your device and get its credentials][lnk-manage-iot-hub]
```

Comment:
why manage hub link?
It should be this one right? https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md#retrieving-user-credentials-to-interact-with-the-service-not-as-a-device

Reply:
this is device sample, so it needs to interact as a device, not service

Reply:
The link you currently have does not seem to point how to provision device or get credentials